_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_golem_ship.dmm

```diff
@@ -0,0 +1,74 @@
+"aa" = (/turf/template_noop,/area/template_noop)
```

Comment:
This will need to be converted to tgm. Feel free to message me on Discord if you need any instructions for that.

Reply:
I think i can figure it out, will do if I need it, thanks.

Reply:
Roger that. Also, don't forget to add the baseturf helper for Lavaland to make sure destroyed turfs get turned into basalt.

Reply:
Its kinda wonky due to being mingled with shuttle code but I'll check that aswell